config/runtime.exs

```diff
@@ -28,12 +28,7 @@ if config_env() == :prod do
   # want to use a different value for prod and you most likely don't want
   # to check this value into version control, so we use an environment
   # variable instead.
-  secret_key_base =
-    System.get_env("SECRET_KEY_BASE") ||
-      raise """
-      environment variable SECRET_KEY_BASE is missing.
-      You can generate one by calling: mix phx.gen.secret
-      """
+  secret_key_base = System.fetch_env!("SECRET_KEY_BASE")
```

Comment:
Let's keep the raise error part because it would help to debug when forget to set env during the deployment

Reply:
OK, I thought as I am using ```fetch_env!``` it would return an error thus I would not need to raise one manually?